fuzzy/docker/base/Dockerfile.ubuntu18.04-verificarlo

```diff
@@ -0,0 +1,43 @@
+FROM ubuntu:18.04
+
+# This Dockerfile is adapted from https://github.com/verificarlo/verificarlo
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update &&\
+    apt install -y autoconf\
+                   automake\
+                   bash ca-certificates\
+                   binutils\
+                   clang-4.0\
+                   gcc-7\
+                   g++-7\
+                   git\
+                   libedit-dev\
+                   libmpfr-dev\
+                   libtool\
+                   libz-dev\
+                   llvm-4.0\
+                   llvm-4.0-dev\
+                   make\
+                   python3\
```

Comment:
Do we need `python3` and the following 2 lines for the basic Verificarlo build? I'd not install it in the "base" because then it makes the subsequent fuzzy python slight more ambiguous, and this slightly more bloated.

Reply:
not sure I understand: do you suggest to use python 2.7 in base?

Reply:
@glatard, do we need python at all for the install? I think I'm saying try to keep it as slim as possible, so would opt for using the OS default python install when it's needed and not adding another.

Reply:
The latest verificarlo does indeed requires python3. (we made the move since some distributions are dropping python2 support.)

Reply:
Ok great, then this is resolved. Thanks!